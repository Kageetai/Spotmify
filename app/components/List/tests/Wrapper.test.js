import React from 'react';
import renderer from 'react-test-renderer';

import Wrapper from '../Wrapper';

describe('<Wrapper />', () => {
  it('should match the snapshot', () => {
    const wrapper = renderer.create(<Wrapper />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
