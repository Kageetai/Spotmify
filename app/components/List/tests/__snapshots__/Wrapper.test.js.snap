// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match the snapshot 1`] = `
.c0 {
  padding: 0;
  margin: 0;
  width: 100%;
  background-color: white;
  border: 1px solid #ccc;
  border-radius: 3px;
  overflow: hidden;
}

<div
  className="c0"
/>
`;
